awx/main/credential_plugins/hashivault.py

```diff
@@ -73,36 +82,55 @@
 hashi_ssh_inputs['required'].extend(['public_key', 'role'])
 
 
+def create_temporary_fifo(data):
```

Comment:
It looks like we've got this function duplicated several places now (conjur, aim, and now here).  Could we place this function in:

`awx/main/utils/common.py`

...and import it here (and in the other plugins, too)?

Reply:
Should be identical, at least the hashivault.py to the one conjur.py - since I just copy&pasted it ;-)